clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/deploy/converters/DeployCloudFoundryServerGroupAtomicOperationConverter.java

```diff
@@ -176,11 +180,9 @@ private ArtifactCredentials getArtifactCredentials(
       Map<Object, Object> manifestMap) {
     List<CloudFoundryManifest> manifestApps =
         new ObjectMapper()
-            .setPropertyNamingStrategy(PropertyNamingStrategy.KEBAB_CASE)
+            .setPropertyNamingStrategy(PropertyNamingStrategies.KEBAB_CASE)
```

Comment:
This looks different than adding a null check. Can you explain what this and the TypeReference change below does?

Reply:
Yup that was just general cleanup while I was in the file - `PropertyNamingStrategy` [was deprecated](https://fasterxml.github.io/jackson-databind/javadoc/2.12/com/fasterxml/jackson/databind/PropertyNamingStrategy.KebabCaseStrategy.html) in favour of `PropertyNamingStrategies`, and the type reference parameter isn't necessary.